### PR TITLE
Adapt code for 3DeepFace 

### PR DESCRIPTION
**Pull request by CunliangGeng**

### CODE REVIEW

File: deeprank/generate/DataGenerator.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __init__(self,pdb_select=None,pdb_source=None,pdb_native=None,pssm_source=No
         # filter the cplx if required
         self.pdb_path = self.all_pdb
         if self.pdb_select is not None:
-            self.pdb_path = list(filter(lambda x: self.pdb_select in x, self.all_pdb))
+            self.pdb_path = list(filter(lambda x: any(map(lambda i: i in x, self.pdb_select)), self.all_pdb))
```
Comment (NicoRenaud) [inline deeprank/generate/DataGenerator.py]: I guess that's working but as a small comment complex one liner should be avoided for readability. I do the same thing though so I know I'm guilty :)

File: deeprank/learn/DataSet.py [hunk 2]
```diff
@@ -601,7 +601,7 @@ def _get_target_ordering(self):
 
         lower_list = ['IRMSD','LRMSD','HADDOCK']
         higher_list = ['DOCKQ','Fnat']
-        NA_list = ['binary_class','BIN_CLASS']
+        NA_list = ['binary_class','BIN_CLASS', 'class']
```
Comment (NicoRenaud) [inline deeprank/learn/DataSet.py]: why do you need a new one there ? And maybe a more self describing name would be good

Comment (CunliangGeng) [reply deeprank/learn/DataSet.py]: What I think is to give a general target name for classification.

File: deeprank/generate/DataGenerator.py (hunk 1 above)
Comment (CunliangGeng) [reply deeprank/generate/DataGenerator.py]: Thanks, let me make it simpler.

**Comment by NicoRenaud** [comment]
It seems that the data generated with that code cannot be read by the learning routines. Try to run all the test (especially the test_learn.py) and let me know. (The test on the master branch are still working)

File: deeprank/generate/DataGenerator.py (hunk 1 above)
Comment (CunliangGeng) [reply deeprank/generate/DataGenerator.py]: Done.

**Comment by CunliangGeng** [comment]
> It seems that the data generated with that code cannot be read by the learning routines. Try to run all the test (especially the test_learn.py) and let me know. (The test on the master branch are still working)

It works well on `liang` branch. What error did you get?

File: deeprank/learn/DataSet.py (hunk 2 above)
Comment (CunliangGeng) [reply deeprank/learn/DataSet.py]: Remove it then, not that necessary.

**Comment by NicoRenaud** [comment]
> > It seems that the data generated with that code cannot be read by the learning routines. Try to run all the test (especially the test_learn.py) and let me know. (The test on the master branch are still working)
> 
> It works well on `liang` branch. What error did you get?

Well it's working now ... I guess I did something wrong last time sorry about that

**Comment by NicoRenaud** [comment]
If you want to keep the branch you can otherwise, we usually open a new branch for every fix that we make.
In that case give them another name (i.e. not Liang :) ) so that the name reflect what you're doing

**Comment by CunliangGeng** [comment]
> If you want to keep the branch you can otherwise, we usually open a new branch for every fix that we make.
> In that case give them another name (i.e. not Liang :) ) so that the name reflect what you're doing

Thanks for the suggestions! Will be more and more professional using git for collaboration :-)

**Pull request merged**